### DISCUSSION TITLE
Nxext 15: Roadmap

### DISCUSSION THREAD

**Discussion by Jordan-Hall** [nxext/nx-extensions#717] [Ideas]
# Version 15 

### @nxext/vite

- [x] Migrate to Vite 3
- [x] Add support for sourcemaps
- [ ] Support eject of vite config
- [x] vitest by default
- [ ] Support plugins via project.json
### @nxext/vitest

- [ ] Create schematics  for all Nxext supported packages
- [ ] InSource schematics  
- [ ] Migrate to InSource 

### @nxext/react

- [ ] Ability to add plugins via project.json to our react builder

### @nxext/storybook (New - planned)

- [ ] migrate from Webpack to Vite builder
- [ ] generators for @nxext plugins

### @nxext/devkit (new)

- [ ] Remove reliant on internal workings of Nx
- [ ] remove dependencies on Angular

###  Others

- [ ] Vitest by default
- [ ] Update to latest version

### Docs

We are working on new documentations and would love help from the community on where we can improve

## Versioning

Currently our versions are pegged to Nx major releases. However, going forward, this will be changed and will no longer be linked to NX Major versions. This allows us to better release features and change our internals workings as we enhance our own devkit

**Comment by hieutranagi47** [comment]
Hi @Jordan-Hall ,
Do you plan to support generating SolidJs as a host or remote in the micro-frontend?
`nx g @nxext/solid:host container --directory=apps --remotes=xxx`
...

**Comment by DominikPieper** [comment]
@hieutranagi47 right now, we don't have any plans around that feature.
